content/java/concepts/math-methods/terms/floor/floor.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Floor'
+Description: 'Returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.'
+Subjects: 
+    -'Code Foundations'
+    -'Computer Science'
+Tags: 
+    - 'Documentation'
+CatalogContent: 
+  - 'learn-java/static-methods-of-the-math-class'
+---
+
+The static function **`.floor()`** included in the `Math` class returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.
+
+## Syntax
+The Math package can be imported and utilized as shown below
```

Comment:
```suggestion
## Syntax

The Math package can be imported and utilized as shown below

```

For headers and code blocks, let's make sure we have an empty newline above _and_ below it. Can we make this edit throughout the rest of this file?

Reply:
Yes, I spaced out the code blocks and headers.